tests/run.sh

```diff
@@ -3,8 +3,11 @@ set -xoeu pipefail
 
 rm -rf outdir
 
-dbspro run -d outdir -f testdata/reads-10k-DBS.fastq.gz \
-    ../construct-info/handles.tsv ../construct-info/ABC-sequences.fasta
+dbspro init testdata/reads-10k-DBS.fastq.gz outdir
 
-m=$(cat outdir/umi-counts.txt | md5sum | cut -f 1 -d" ")
-test $m == 62ce11e3ac4fb7428a6bf4bfed84f529
\ No newline at end of file
+pushd outdir
+
+dbspro run
+
+m=$(cat umi-counts.txt | sort | md5sum | cut -f 1 -d" ")
+test $m == 3c31f9634af4238cb0037170104e6d5e
```

Comment:
Did you check into why the md5sum changed? I would assume output should be the same after implemented this change.

Reply:
It was only because the order changed. Running the command above gave the exact same md5sum.

I believe the commit message is ~"make the md5sum order independent".